platform-core-module/build.gradle

```diff
@@ -8,7 +8,7 @@ dependencies {
 	api "com.sun.mail:mailapi:$versions.sunMail"
 	api "com.sun.mail:smtp:$versions.sunMail"
 
-	implementation("jcifs:jcifs:$versions.jcifs") {
+	implementation("eu.agno3.jcifs:jcifs-ng:$versions.jcifsNg") {
 		exclude group: 'javax.servlet', module: 'servlet-api'
```

Comment:
do we still need this exclusion?

Reply:
I have no idea, dont really know what it does and how to test it when I remove it

Reply:
Well, just try to remove it. Then:
- `./gradlew allDependencies | grep FAIL` must yield no output
- `./gradlew clean check` must succeed
- manual smb tests must still work

Reply:
works, so removed it